ecosystem/sep-0006.md

```diff
@@ -86,11 +86,11 @@ sequenceDiagram
     Anchor-->>-Wallet: transaction id
 
     note right of Wallet: If User has already <br> been accepted, <br> KYC is unnecessary.
-    Wallet->>+Anchor: GET [SEP-12]/customer?transaction_id=<id>&type=
```

Comment:
Type should still be passed. Theoretically anchor can use numerical ids for transactions (by type), therefore it'd need to distinguish between sep-6 transaction id N and SEP-31.
In addition, it makes implementation on AP more straightforward (otherwise we need to query both tables to find transaction with this ID)

Reply:
Sure I can add it back. Just curious, what should the customer type be for SEP-6? Currently, there is no configuration to define a SEP-6 customer type

Reply:
Should just pass types as sep-6 and sep-31. We can make recommendation in the standard to follow this rule, but that's generally how AP should be implemented to use the correct table